libsplinter/src/store/postgres.rs

```diff
@@ -120,4 +120,15 @@ impl StoreFactory for PgStoreFactory {
     fn get_biome_user_profile_store(&self) -> Box<dyn crate::biome::UserProfileStore> {
         Box::new(crate::biome::DieselUserProfileStore::new(self.pool.clone()))
     }
+
+    #[cfg(feature = "admin-service-event-store")]
```

Comment:
This needs to have a method for when the “admin-service-event-store-diesel” is enabled an one for when it isn’t, same with the SQLite version.

Reply:
Updated with `unimplemented!()` in these cases